app/TuistApp/Sources/Views/DevicesView/DevicesViewModel.swift

```diff
@@ -56,22 +56,43 @@ struct PinnedSimulatorsKey: AppStorageKey {
 
 struct SelectedDeviceKey: AppStorageKey {
     static let key = "selectedSimulator"
-    static let defaultValue: SelectedDevice? = nil
+    static let defaultValue: ReducedDevice? = nil
 }
 
 enum SelectedDevice: Codable, Equatable {
     case simulator(SimulatorDeviceAndRuntime)
     case device(PhysicalDevice)
 }
 
+enum ReducedDevice: Codable, Equatable {
+    case simulator(id: String)
+    case device(id: String)
```

Comment:
```suggestion
fileprivate enum SelectedDevice: Codable, Equatable {
    case simulator(id: String)
    case device(id: String)
```

If we rename `SelectedDevice` to `Device`, we can call this one `SelectedDevice`. We can keep it `fileprivate`.

Reply:
I’ll rename this to `SelectedDevice`, but since it’s referenced by `AppStorageKey`, it should be set to `internal`.

Reply:
gotcha, yeah, that's fine then 👍